pandas/tests/frame/test_rank.py

```diff
@@ -257,43 +264,38 @@ def _check2d(df, expected, method='average', axis=0):
             result = df.rank(method=method, axis=axis)
             assert_frame_equal(result, exp_df)
 
-        dtypes = [None, object]
         disabled = set([(object, 'first')])
-        results = self.results
-
-        for method, axis, dtype in product(results, [0, 1], dtypes):
-            if (dtype, method) in disabled:
-                continue
-            frame = df if dtype is None else df.astype(dtype)
-            _check2d(frame, results[method], method=method, axis=axis)
-
-
-@pytest.mark.parametrize(
-    "method,exp", [("dense",
-                    [[1., 1., 1.],
-                     [1., 0.5, 2. / 3],
-                     [1., 0.5, 1. / 3]]),
-                   ("min",
-                    [[1. / 3, 1., 1.],
-                     [1. / 3, 1. / 3, 2. / 3],
-                     [1. / 3, 1. / 3, 1. / 3]]),
-                   ("max",
-                    [[1., 1., 1.],
-                     [1., 2. / 3, 2. / 3],
-                     [1., 2. / 3, 1. / 3]]),
-                   ("average",
-                    [[2. / 3, 1., 1.],
-                     [2. / 3, 0.5, 2. / 3],
-                     [2. / 3, 0.5, 1. / 3]]),
-                   ("first",
-                    [[1. / 3, 1., 1.],
-                     [2. / 3, 1. / 3, 2. / 3],
-                     [3. / 3, 2. / 3, 1. / 3]])])
-def test_rank_pct_true(method, exp):
```

Comment:
This test was the only one in that was outside the class, which seemed a bit strange, so I moved it inside the class.